### PR TITLE
fix: OPF unit test coverage opf-checkout-billing-address-component-service

### PR DESCRIPTION
**Pull request by kpawelczak**

### CODE REVIEW

File: integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts [hunk 1]
```diff
@@ -164,14 +164,14 @@ export class OpfCheckoutBillingAddressFormService {
     this.isSameAsDeliverySub.next(value);
   }
 
-  protected getDeliveryAddress(): Observable<Address | undefined> {
+  getDeliveryAddress(): Observable<Address | undefined> {
```
Comment (rmch91) [inline integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts]: Is the access modifiers were changed only for using this in spec? If yes, i don't think it's a good idea.
I think will be better to access this methods with `service['getDeliveryAddress']()` or to use `service as any` in spy if needed

File: integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts [hunk 2]
```diff
@@ -164,14 +164,14 @@ export class OpfCheckoutBillingAddressFormService {
...
     return this.checkoutDeliveryAddressFacade.getDeliveryAddressState().pipe(
       filter((state) => !state.loading),
       map((state) => state.data)
     );
   }
 
-  protected getPaymentAddress(): Observable<Address | undefined> {
+  getPaymentAddress(): Observable<Address | undefined> {
```
Comment (rmch91) [inline integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts]: as above

**Review by rmch91 — CHANGES_REQUESTED** [review]

File: integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts (hunk 1 above)
Comment (kpawelczak) [reply integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts]: fixed

File: integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts (hunk 2 above)
Comment (kpawelczak) [reply integration-libs/opf/checkout/components/opf-checkout-billing-address-form/opf-checkout-billing-address-form.service.ts]: fixed

**Review by rmch91 — APPROVED** [review]

**Pull request merged**